dpctl/_sycl_platform.pyx

```diff
@@ -366,6 +375,78 @@ cdef class SyclPlatform(_SyclPlatform):
         """
         return DPCTLPlatform_Hash(self._platform_ref)
 
+    def get_devices(self, device_type=device_type_t.all):
+        """
+        Returns the list of :class:`dpctl.SyclDevice` objects associated with
+        :class:`dpctl.SyclPlatform` instance selected based on
+        the given :class:`dpctl.device_type`.
```

Comment:
A link to `device_type` is not available in the documentation.

Reply:
It appears the enum types defined in `enum_types.py` are not documented anywhere. They will need a page similar to the constants page for `dpctl.tensor`

Good catch!

Reply:
I will add that in a separate PR, for the sake of not over-loading this one